sdk/src/main/java/com/kin/ecosystem/marketplace/view/MarketplaceFragment.java

```diff
@@ -123,7 +124,7 @@ public void onItemClick(BaseRecyclerAdapter adapter, View view, int position) {
 
 	@Override
 	public void setSpendList(List<Offer> spendList) {
-		spendRecyclerAdapter.setNewData(spendList);
+//		spendRecyclerAdapter.setNewData(spendList);
```

Comment:
Same comment as above, if no need for this method them delete it,

Reply:
it was only for testing, next task is handling all of this so, I didn't bother.